pennylane_lightning_gpu/src/util/MPIManager.hpp

```diff
@@ -533,6 +542,41 @@ class MPIManager final {
         return recvBuf;
     }
 
+    /**
+     * @brief MPI_Send wrapper.
+     *
+     * @tparam T C++ data type.
+     * @param sendBuf Send buffer vector.
+     * @param dest Rank of send dest.
+     */
+
+    template <typename T> void Send(std::vector<T> &sendBuf, size_t dest) {
+        MPI_Datatype datatype = getMPIDatatype<T>();
+        const int tag = 0;
```

Comment:
Do we ever need to set tag to a different value?

Reply:
I've often just sent messages with `tag=0`, since I have never had a use for it. I would suggest using `MPI_ANY_TAG` to make it clear it does not matter.

Reply:
Updated with `MPI_ANY_TAG`